drivers/sqlite-driver/src/main/kotlin/app/cash/sqldelight/driver/jdbc/sqlite/JdbcSqliteDriver.kt

```diff
@@ -14,8 +13,24 @@ import kotlin.concurrent.getOrSet
 @Suppress("DELEGATED_MEMBER_HIDES_SUPERTYPE_OVERRIDE")
 class JdbcSqliteDriver constructor(
   /**
-   * Database connection URL in the form of `jdbc:sqlite:path` where `path` is either blank
-   * (creating an in-memory database) or a path to a file.
+   * Database connection URL in the form of `jdbc:sqlite:path?key1=value1&...` where:
+   * - `jdbc:sqlite:` is the prefix which instructs [DriverManager] to open a connection
+   *   using the provided [org.sqlite.JDBC] Driver.
+   * - `path` is a file path which instructs sqlite *where* it should open the database
+   *   connection.
+   * - `?key1=value1&...` is an optional query string which instruct sqlite *how* it
+   *   should open the connection.
+   *
+   * Examples:
+   * - `jdbc:sqlite:/path/to/myDatabase.db` opens a database connection, writing changes
+   *   to the filesystem at the specified `path`.
+   * - `jdbc:sqlite:` (i.e. an empty path) will create a temporary database whereby the
+   *   temp file is deleted upon connection closure.
+   * - `jdbc:sqlite::memory:` will create a purely in-memory database.
+   * - `jdbc:sqlite:memdb1?mode=memory&cache=shared` will create a named in-memory
```

Comment:
This should actually be `jdbc:sqlite:file:memdb1?mode=memory&cache=shared`.

SQLite would otherwise create a file in the current working directory...

Reply:
Resolved in commit d2e1ead